### ISSUE TITLE
Iceberg support 

### ISSUE THREAD

**Issue by pixie79** [redpanda-data/connect#2624]
Are there plans for an Iceberg output with AWS Glue Catalog support? (Or multiple iceberg catalogs support)

This would be useful for tools like Athena to then be able to query micro delay realtime data.

**Comment by hendoxc** [comment]
Agree. A that a large reason why I'd still use Kafka Connect is for the Iceberg Sink Connector. Given that iceberg has become the industry standard for many orgs storing data in s3/gcs/azure.

**Comment by twmb** [comment]
An Iceberg output has been added to Redpanda Connect. Closing as implemented. If you encounter issues with the Iceberg output, feel free to open a new issue or PR.
